website/developer-docs/docs/templates/procedural.md

```diff
@@ -3,16 +3,16 @@ title: "Procedural topic"
 ---
 
 :::info
-**How to use this template**: start with the [markdown version](./procedural.tmpl.md) of the template, either copying the file from the local repo or download the template file using the following command:
+**How to use this template**: start with the markdown version of the template, either by copying the [`procedural.tmpl.md`](https://github.com/goauthentik/authentik/tree/main/website/developer-docs/docs/templates) file from our GitHub repo or downloading the template file using the following command:
 
 ```
-wget https://raw.githubusercontent.com/goauthentik/authentik/main/website/developer-docs/docs/templates/templates/procedural.tmpl.md
+wget https://github.com/goauthentik/authentik/blob/main/website/developer-docs/docs/templates/procedural.tmpl.md
```

Comment:
```suggestion
wget https://raw.githubusercontent.com/goauthentik/authentik/main/website/developer-docs/docs/templates/procedural.tmpl.md
```

running `wget` directly against github.com is gonna return HTML and not the raw file

Reply:
ach. ok, now I understand the use of the `raw.githubusercontent`. The first link did not work (but this one you just added does). Here's error from the first link:
`tanaberry@Tanas-MacBook-Pro-authentik Documents % wget https://raw.githubusercontent.com/goauthentik/authentik/main/website/developer-docs/docs/templates/templates/procedural.tmpl.md
--2023-09-06 15:40:25--  https://raw.githubusercontent.com/goauthentik/authentik/main/website/developer-docs/docs/templates/templates/procedural.tmpl.md
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 2606:50c0:8002::154, 2606:50c0:8000::154, 2606:50c0:8003::154, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|2606:50c0:8002::154|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-09-06 15:40:25 ERROR 404: Not Found`

Reply:
ok,let me go fix the other two wgets then... be right back